### PR TITLE
add data to deps.edn

### PR DESCRIPTION
**Pull request by the-dress-code**
I added 2 keys to the deps.edn. We'll fill them in later.

### CODE REVIEW

File: src/katas/arg-extract1.clj [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  "Write a function takes a vector as an argument, `arg`
+   extracts every other value and places it into a new vector. "
+  [arg]
+  (->> arg       ;; "->>" is the thread-last macro
+       (cons 0)
+       (take-nth 2)
+       rest
+       vec))
+
+(comment
```
Comment (lambduhh) [inline src/katas/arg-extract1.clj:13]: So usually it's best practice to delete the comment block.

**Review by patclanett — APPROVED** [review]
I don't like the semicolons in the comment >:(

**Review by lambduhh — APPROVED** [review]
lgtm!

File: src/katas/arg-extract1.clj (hunk 1 above)
Comment (the-dress-code) [reply src/katas/arg-extract1.clj:13]: I know Rich Hickey personally, and he approves of my usage.

**Pull request merged**